src/pudl/transform/phmsagas.py

```diff
@@ -0,0 +1,159 @@
+        "operator_id_phmsa",
+        "operator_name_phmsa",
+        "office_address_street",
+        "office_address_city",
+        "office_address_state",
+        "office_address_zip",
+        "office_address_county",
+        "headquarters_address_street",
+        "headquarters_address_city",
+        "headquarters_address_state",
+        "headquarters_address_zip",
+        "headquarters_address_county",
+        "excavation_damage_excavation_practices",
+        "excavation_damage_locating_practices",
+        "excavation_damage_one_call_notification",
+        "excavation_damage_other",
+        "excavation_damage_total",
+        "excavation_tickets",
+        "services_efv_in_system",
+        "services_efv_installed",
+        "services_shutoff_valve_in_system",
+        "services_shutoff_valve_installed",
+        "federal_land_leaks_repaired_or_scheduled",
+        "percent_unaccounted_for_gas",
+        "additional_information",
+        "preparer_email",
+        "preparer_fax",
+        "preparer_name",
+        "preparer_phone",
+        "preparer_title",
+    ],
+    "columns_to_convert_to_ints": [
+        "report_year",
+        "report_number",
+        "operator_id_phmsa",
+        "excavation_damage_excavation_practices",
+        "excavation_damage_locating_practices",
+        "excavation_damage_one_call_notification",
+        "excavation_damage_other",
+        "excavation_damage_total",
+        "excavation_tickets",
+        "services_efv_in_system",
+        "services_efv_installed",
+        "services_shutoff_valve_in_system",
+        "services_shutoff_valve_installed",
+    ],
+    "capitalization_exclusion": ["headquarters_address_state", "office_address_state"],
+}
+
+##############################################################################
+# PHMSAGAS transformation logic
+##############################################################################
+
+
+@asset(
+    ins={"raw_data": AssetIn(key="raw_phmsagas__yearly_distribution")},
+    io_manager_key=None,  # TODO: check on this one ("pudl_io_manager"? something else?)
+    compute_kind=None,  # TODO: check on this one
+)
+def core_phmsagas__yearly_distribution_operators(
+    raw_data: pd.DataFrame,
+) -> pd.DataFrame:
+    """Build the :ref:`core_phmsagas__yearly_distribution_operators`."""
+    df = raw_data.loc[
+        :, YEARLY_DISTRIBUTION_OPERATORS_COLUMNS["columns_to_keep"]
+    ].copy()
+
+    # Standardize NAs
+    df = standardize_na_values(df)
+
+    # Initial string cleaning
+    for col in df.select_dtypes(include=["object"]).columns:
+        df[col] = df[col].str.strip()
+
+    # Specify the columns to convert to integer type
+    cols_to_convert = YEARLY_DISTRIBUTION_OPERATORS_COLUMNS[
+        "columns_to_convert_to_ints"
+    ]
+
+    # Fill NaN values with pd.NA, then cast to "Int64" nullable integer type
+    df[cols_to_convert] = df[cols_to_convert].fillna(pd.NA).astype("Int64")
```

Comment:
Non-blocking: `convert_dtypes()` might more simply achieve what you're trying to do here, since it handles the conversion of nulls to Pandas nullable dtypes.

Reply:
Updated